_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,160 @@
+---
+id: 24afc2
+name: Letter spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that the letter spacing can be adjusted by the user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][], for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+
+## Expectation 1
+
+For the test target's [letter-spacing][] property, one of the following is true:
```

Comment:
This conditional list doesn't work well. You can see it if you try to declare what this list is about in each item, which is required in conditional lists:

```
- the [letter-spacing][] property has a [declared][] value specified via the `style` attribute does not have the [important flag][]; or
- the [letter-spacing][] property has a [computed][] value is at least `0.12` times its [computed][] [font-size][].
```

These two items are not about the same thing. One is about what's declared in the attribute, the other is about the computed property. I think the way this should be done is to write them out in a sentence, where the first thing is the requirement, and the second is an exception. Something like this:

```suggestion
For each test target, there is no [important flag][] on the [declared][] value of the [letter-spacing][] property in the `style` attribute, except if the [computed][] value is at least `0.12` times its [computed][] [font-size][].
```

Reply:
I disagree. I think that "except" clauses read poorly and require more mental effort to understand than "or" clauses.
"A except if B" is the same logical construction as "A or B". Just written in a more convoluted way.

Reply:
I must agree with @Jym77 on that. In fact, I've rewritten all such exceptions into opposite form for our internal versions of the rules as it makes things so much easier to digest.

Reply:
Agree the current 'or' version is easier to read, though undecided if that is because of the use of the 'or' or the use of bullets. That said, bullets make for easier reading and probably work better with an 'or'.

Reply:
That doesn't really change that this condition list doesn't include the subject; that it can't because it is about two different things.

Reply:
The subject is the test target's letter-spacing property, so both declared value and computed value can be relevant for that.

Am I missing something?

Reply:
Does this work...

For the test target, one of the following is true:

- the [declared][] value of the test target's [letter-spacing][] property, specified via the 'style' attribute, does not have the important flag; or
- the [computed][] value of the test target's [letter-spacing][] property is at least `0.12` times its [computed][] [font-size][].

Reply:
I agree that it is easier to read the 'or' version. And I was going to suggest exactly the same that @EmmaJP did

Reply:
If we absolutely want to repeat and frontload something:

> For the test target [letter-spacing][] property, one of the following is true:
> - the property has a [declared][] value specified via the `style` attribute without the important flag; or
> - the property has a [computed][] value of at least `0.12` times its [computed][] [font-size][].

But I don't think it reads really better. Notably because:
> For the test target [letter-spacing][] property, one of the following is true:
> - its [declared][] value specified via the `style` attribute does not have the important flag; or
> - its [computed][] value is at least `0.12` times its [computed][] [font-size][].

is more readable imho. Or @EmmaJP suggestion.